server/keyshare/keyshareserver/server.go

```diff
@@ -178,6 +186,56 @@ func (s *Server) loadIdemixKeys(conf *irma.Configuration) error {
 	return errs.ErrorOrNil()
 }
 
+// /prove/getPs
+func (s *Server) handlePs(w http.ResponseWriter, r *http.Request) {
+	// Fetch from context
+	user := r.Context().Value("user").(*User)
+	authorization := r.Context().Value("authorization").(string)
+
+	// Read keys
+	var keys []irma.PublicKeyIdentifier
+	if err := server.ParseBody(r, &keys); err != nil {
+		server.WriteError(w, server.ErrorInvalidRequest, err.Error())
+		return
+	}
+	if len(keys) == 0 {
+		s.conf.Logger.Info("Malformed request: no keys for P specified")
+		server.WriteError(w, server.ErrorInvalidRequest, "No key specified")
```

Comment:
In Go, error messages like `No key specified` are expected in lowercase. This suits better the way how errors are printed in logging. I know we do not do this consistently, but maybe a good practice to not make it worse.

https://earthly.dev/blog/golang-errors/